allocations/allocations_api.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import json
+from django.conf import settings
+import logging
+from keystoneauth1.identity import v3
+from keystoneauth1 import session
+
+logger = logging.getLogger('default')
+
+class BALANCE_SERVICE_API:
+
+    def call( self, project_code ):
+        # get token
+        auth = v3.Password(auth_url=settings.OPENSTACK_KEYSTONE_URL,
+                           username=settings.OPENSTACK_SERVICE_USERNAME,
+                           password=settings.OPENSTACK_SERVICE_PASSWORD,
+                           project_id=settings.OPENSTACK_SERVICE_PROJECT_ID,
+                           user_domain_name='default',
+                           user_domain_id='default')
+        sess = session.Session(auth=auth)
+        auth_token = sess.get_token()
+        
+        headers = {'Accept': 'appliaction/json',
+                   'X-Auth-Token': auth_token,
+                   'Content-Type': 'appliaction/json'}
+        
+        url = self.make_url( project_code )
+        logger.info('Requesting %s from allocations balance service ...', url)
+        resp = requests.get(url, headers=headers)
+        logger.info('Response received. Parsing to json ...')
+        
+        data = json.loads(resp.text)
+        return data
+
+    def make_url( self, project_code ):
+        return '{0}/{1}'.format( settings.ALLOCATIONS_BALANCE_SERVICE_ROOT_URL, project_code)
```

Comment:
What if ALLOCATIONS_BALANCE_SERVICE_ROOT_URL is `None` or empty string? Or otherwise malformed? Where does it make sense to raise that error?